nixos/modules/services/networking/corerad.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.corerad;
+in {
+  meta = {
+    maintainers = with maintainers; [ mdlayher ];
+  };
+
+  options.services.corerad = {
+    enable = mkEnableOption "CoreRAD IPv6 NDP RA daemon";
+
+    configFile = mkOption {
+      type = types.path;
+      example = "${pkgs.corerad}/etc/corerad/corerad.toml";
```

Comment:
This should be
```nix
{
  example = literalExample "\${pkgs.corerad}/etc/corerad/corerad.toml";
}
```

Reply:
Or actually, I guess this is even better:
```nix
{
  example = literalExample "\"\${pkgs.corerad}/etc/corerad/corerad.toml\"";
}
```

We really need some better guidelines for this..

Reply:
Thanks for the quick review. Updated.

Reply:
Ack, will revise.